src/main/java/org/bitbuckets/drive/controlsds/DriveControl.java

```diff
@@ -77,22 +77,6 @@ public void stop() {
         });
     }
 
-    public void stopSticky() {
-        doDriveWithStates(new SwerveModuleState[]{
-                new SwerveModuleState(0, Rotation2d.fromDegrees(45)),
-                new SwerveModuleState(0, Rotation2d.fromDegrees(-45)),
-                new SwerveModuleState(0, Rotation2d.fromDegrees(-45)),
-                new SwerveModuleState(0, Rotation2d.fromDegrees(45))
-        });
-    }
-
-    public void stopGentle() {
-        for (int i = 0; i < 4; i++) {
-            modules.get(i).stopMotor();
-
-        }
-    }
```

Comment:
this functionality was added to set the motors to 0 output so they didn't try and move to a position. I think we still want it.